models/basgra/README.md

```diff
@@ -0,0 +1,64 @@
+A generic template for adding a new model to PEcAn
```

Comment:
For this PR, I would just remove this file altogether. Eventually, it would be nice to make a model-specific README with a brief explanation of what this is and links to additional documentation.